src/main/java/net/sf/jabref/gui/journals/ManageJournalAbbreviationsController.java

```diff
@@ -0,0 +1,7 @@
+package net.sf.jabref.gui.journals;
+
+/**
+ * Created by Tobias on 13.09.2016.
+ */
+public class ManageJournalAbbreviationsController {
```

Comment:
As described in #1918 somehow the manageJournalAbbreviations dialog made some trouble for the tests. running the tests locally gave me stacktrace below. So after some trial and error and investigation I came to the conclusion that the only differenec for the journal abbreviations dialog is that the controller specified in the fxml file is also the View class. After creating this class and setting it as the controller the error message is gone.
<details>
<summary>Stacktrace for failed test</summary>

```
net.sf.jabref.logic.l10n.LocalizationConsistencyTest > findObsoleteLocalizationKeys FAILED
    java.lang.VerifyError: Bad type on operand stack
    Exception Details:
      Location:
        net/sf/jabref/gui/journals/ManageJournalAbbreviationsView.<init>()V @31: putfield
      Reason:
        Type 'net/sf/jabref/logic/l10n/LocalizationBundle' (current frame, stack[1]) is not assignable to 'java/util/ResourceBundle'
      Current Frame:
        bci: @31
        flags: { }
        locals: { 'net/sf/jabref/gui/journals/ManageJournalAbbreviationsView', '[Z' }
        stack: { 'net/sf/jabref/gui/journals/ManageJournalAbbreviationsView', 'net/sf/jabref/logic/l10n/LocalizationBundle' }
      Bytecode:
        0x0000000: b802 cb4c 2ab7 0006 2b03 0454 2abb 0007
        0x0000010: 59b7 0008 b500 032b 0404 542a b800 09b5
        0x0000020: 000a 2b05 0454 b1
        at java.lang.Class.getDeclaredConstructors0(Native Method)
        at java.lang.Class.privateGetDeclaredConstructors(Class.java:2671)
        at java.lang.Class.getConstructor0(Class.java:3075)
        at java.lang.Class.newInstance(Class.java:412)
        at sun.reflect.misc.ReflectUtil.newInstance(ReflectUtil.java:51)
        at javafx.fxml.FXMLLoader$ValueElement.processAttribute(FXMLLoader.java:927)
        at javafx.fxml.FXMLLoader$InstanceDeclarationElement.processAttribute(FXMLLoader.java:971)
        at javafx.fxml.FXMLLoader$Element.processStartElement(FXMLLoader.java:220)
        at javafx.fxml.FXMLLoader$ValueElement.processStartElement(FXMLLoader.java:744)
        at javafx.fxml.FXMLLoader.processStartElement(FXMLLoader.java:2707)
        at javafx.fxml.FXMLLoader.loadImpl(FXMLLoader.java:2527)
        at javafx.fxml.FXMLLoader.loadImpl(FXMLLoader.java:2441)
        at javafx.fxml.FXMLLoader.load(FXMLLoader.java:2409)
        at net.sf.jabref.logic.l10n.LocalizationParser.getLanguageKeysInFxmlFile(LocalizationParser.java:173)
        at net.sf.jabref.logic.l10n.LocalizationParser.lambda$findLocalizationEntriesInFxmlFiles$2(LocalizationParser.java:90)
        at java.util.stream.ReferencePipeline$7$1.accept(ReferencePipeline.java:267)
        at java.util.stream.ReferencePipeline$2$1.accept(ReferencePipeline.java:175)
        at java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:193)
        at java.util.Iterator.forEachRemaining(Iterator.java:116)
        at java.util.Spliterators$IteratorSpliterator.forEachRemaining(Spliterators.java:1801)
        at java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:481)
        at java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:471)
        at java.util.stream.ReduceOps$ReduceOp.evaluateSequential(ReduceOps.java:708)
        at java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
        at java.util.stream.ReferencePipeline.collect(ReferencePipeline.java:499)
        at net.sf.jabref.logic.l10n.LocalizationParser.findLocalizationEntriesInFxmlFiles(LocalizationParser.java:91)
        at net.sf.jabref.logic.l10n.LocalizationParser.findLocalizationEntriesInFiles(LocalizationParser.java:73)
        at net.sf.jabref.logic.l10n.LocalizationParser.findObsolete(LocalizationParser.java:52)
        at net.sf.jabref.logic.l10n.LocalizationConsistencyTest.findObsoleteLocalizationKeys(LocalizationConsistencyTest.java:151)
```

</details>